The job **`mingw-check`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/7171454535/job/19526499859) [(plain)](https://github.com/rust-lang/rust/commit/aaf2519e3bbddddab47fd49866a14b10b9c0deae/checks/19526499859/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
GITHUB_ACTION=__run_7
GITHUB_ACTIONS=true
GITHUB_ACTION_REF=
GITHUB_ACTION_REPOSITORY=
GITHUB_ACTOR=haydonryan
GITHUB_API_URL=https://api.github.com
GITHUB_BASE_REF=master
GITHUB_ENV=/home/runner/work/_temp/_runner_file_commands/set_env_244382f6-76a0-4574-9b3a-22315b682cc9
GITHUB_EVENT_NAME=pull_request
---
GITHUB_SERVER_URL=https://github.com
GITHUB_SHA=b8122f3ca96797e90513fdf6faa02e3229c93e5b
GITHUB_STATE=/home/runner/work/_temp/_runner_file_commands/save_state_244382f6-76a0-4574-9b3a-22315b682cc9
GITHUB_STEP_SUMMARY=/home/runner/work/_temp/_runner_file_commands/step_summary_244382f6-76a0-4574-9b3a-22315b682cc9
GITHUB_TRIGGERING_ACTOR=haydonryan
GITHUB_WORKFLOW_REF=rust-lang/rust/.github/workflows/ci.yml@refs/pull/118623/merge
GITHUB_WORKFLOW_SHA=b8122f3ca96797e90513fdf6faa02e3229c93e5b
GITHUB_WORKSPACE=/home/runner/work/rust/rust
GOROOT_1_19_X64=/opt/hostedtoolcache/go/1.19.13/x64
---
...........................iiiii..................................................i..... 1144/1158
..............

failures:
error: doctest failed, to rerun pass `-p std --doc`
---- library/std/src/fs.rs - fs::read_to_string (line 284) stdout ----
error[E0405]: cannot find trait `Error` in this scope
##[error] --> library/std/src/fs.rs:287:33
Build completed unsuccessfully in 0:04:58
```

</details>

--- a/library/std/src/fs.rs
+++ b/library/std/src/fs.rs
@@ -242,10 +242,10 @@ pub struct DirBuilder {
 ///
 /// ```no_run
 /// use std::fs;
-/// use std::net::SocketAddr;
 ///
 /// fn main() -> Result<(), Box<dyn std::error::Error + 'static>> {
-///     let foo: SocketAddr = String::from_utf8_lossy(&fs::read("address.txt")?).parse()?;
+///     let address: String = String::from_utf8_lossy(&fs::read("address.txt")?).to_string();
+///     println!("{}", address);
 ///     Ok(())
 /// }
 /// ```

